introduction.md

```diff
@@ -0,0 +1 @@
+Hi! My name is Yash and I'm trying to complete the Mastering Github challenge.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.